mscore/CMakeLists.txt

```diff
@@ -682,9 +682,9 @@ endif (APPLE)
 
 # Add the appropriate splash screen for the build
 if (MSCORE_UNSTABLE)
-    configure_file(/assets/splash-dev.png  mscore/data/splash.png COPYONLY) # Dev splash on development builds
+    configure_file(../assets/splash-dev.png  mscore/data/splash.png COPYONLY) # Dev splash on development builds
```

Comment:
@liamrosenfeld, somebody has programmed things a bit oddly, so as it happens you do need to copy the icons into the `${PROJECT_SOURCE_DIR}`. This code will do that, and will also work for nightlies:

```cmake
# Add the appropriate splash screen for the build
if (MSCORE_UNSTABLE)
    if ("$ENV{BINTRAY_REPO_OWNER}" STREQUAL "musescore")
        # This is a nightly build
        configure_file(../assets/splash-nightly.png ${PROJECT_SOURCE_DIR}/mscore/data/splash.png COPYONLY)
    else ("$ENV{BINTRAY_REPO_OWNER}" STREQUAL "musescore")
        # This is someone building on their own
        configure_file(../assets/splash-dev.png ${PROJECT_SOURCE_DIR}/mscore/data/splash.png COPYONLY)
    endif ("$ENV{BINTRAY_REPO_OWNER}" STREQUAL "musescore")
else (MSCORE_UNSTABLE)
    # Standard splash on stable releases
    configure_file(../assets/splash-stable.png ${PROJECT_SOURCE_DIR}/mscore/data/splash.png COPYONLY)
endif (MSCORE_UNSTABLE)
```

As you can see, I suggest moving the nightly splash to be with the others. In fact, you should ideally put your Illustrator document in `/assests` and move the PNGs into `/assets/generated`.

Now that the splash image is being copied into the source directory it will be tracked by Git, which could become very annoying. To prevent this, you need to add this line to the file [`/.gitignore`](https://github.com/musescore/MuseScore/blob/c7fa7c6fc6f2c53b6475925555705da00eb78a07/.gitignore#L37):

```gitignore
/mscore/data/splash.png
```

Yes, that's an absolute path!

Once that line is added you need to delete the image at `mscore/data/splash.png` **before** you commit. Now the splash icon will be copied into the right place at compile time, and it will be ignored by Git.

Reply:
Thanks, that looks great! When we move the nightly splash with the others should I update the Travis build with the new location or just keep two instances?

Reply:
Also, it appears that Illustrator and Photoshop files are not compatible with diff. So I'll just remove them.

Reply:
Also, do you think we should put the pngs and svgs in /assets/generated or the Illustrator and photoshop in /assets/originals?